protocols/virtualfund/virtualfund.go

```diff
@@ -98,6 +99,11 @@ func (c *Connection) Funded() bool {
 // getExpectedGuarantee returns a map of asset addresses to guarantees for a Connection.
 func (c *Connection) getExpectedGuarantee() consensus_channel.Guarantee {
 	amountFunds := c.GuaranteeInfo.LeftAmount.Add(c.GuaranteeInfo.RightAmount)
+
+	//HACK: GuaranteeInfo stores amounts as types.Funds.
```

Comment:
This is beyond the scope of this PR but I think we should decide how we want to treat multi-asset outcomes throughout our codebase.  Right now some code supports multi-asset (but is not fully tested) and some does not.  It would be nice if we could be consistent in our approach.

Reply:
https://github.com/statechannels/go-nitro/issues/516